rllib/examples/attention_net.py

```diff
@@ -195,12 +195,7 @@ def get_cli_args():
             total_reward = 0
             # start with all zeros as state
             num_transformers = config["model"]["attention_num_transformer_units"]
-            attention_dim = config["model"]["attention_dim"]
-            memory = config["model"]["attention_memory_inference"]
-            init_state = state = [
-                np.zeros([memory, attention_dim], np.float32)
-                for _ in range(num_transformers)
-            ]
+            state = algo.get_policy().get_initial_state()
```

Comment:
This change is only for consistency of the initial state. Previously, we did not have a proper initial state for the attention wrappers but it did not matter because there was no agent collector that needed to sample from it.